Sources/KituraWebSocket/WSConnectionUpgradeFactory.swift

```diff
@@ -38,6 +41,26 @@ public class WSConnectionUpgradeFactory: ProtocolHandlerFactory {
         return connection
     }
 
+    // Return all the extension handlers enabled for this connection
+    public func extensionHandlers(header: String) -> [ChannelHandler] {
```

Comment:
If this needs to be public we should have /// for jazzy docs. If this doesn't need to be public we can remove `public`.